clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -42,7 +42,11 @@
 import org.apache.kafka.clients.consumer.internals.events.ApplicationEventProcessor;
```

Comment:
Do we have any new tests to cover the changes in this class?

Reply:
I added three tests to `AsyncKafkaConsumerTest` to test the `processBackgroundEvents` method.